src/main/scala/net/manub/embeddedkafka/EmbeddedKafka.scala

```diff
@@ -239,4 +246,8 @@ sealed trait EmbeddedKafkaSupport {
     broker.startup()
     broker
   }
+
+  def createCustomTopic(topic: String, topicConfig: Properties)(implicit config: EmbeddedKafkaConfig): Unit = {
```

Comment:
Working with Properties in Scala is a bit ugly, have you considered using a `Map[String, String` for the interface of this function instead ?